Add Safe keyboard detector for android (#1)
bamlab/react-native-app-security: README.md

]
```

## Safe Keyboard Detector

> ⚠ Warning : This functionality is ONLY implemented on Android since iOS already prevent the use of third party keyboard on sensitive filed such as passwords. 

Reviewer: ```suggestion
⚠ Warning : This functionality is ONLY implemented on Android since iOS already prevents the use of third party keyboards on sensitive fields such as passwords. 
```

> **🥷 What's the threat?** Third party keyboard might embed a malicious keylogger program to record passwords and sensitive data [More details](https://www.synopsys.com/blogs/software-security/mitigate-third-party-mobile-keyboard-risk.html)

How to mitigate this threat : 

> - Compare the current keyboard id with a list of known keyboard package names.

Reviewer: ```suggestion
- Compare the current keyboard id with a list of known keyboard package names that we deem safe (keyboards by google, samsung, microsoft).
```

- Display the android native input method picker for the user to change its keyboard.

```tsx
import { SafeKeyboardDetector } from '@bam.tech/react-native-app-security';
 
  const isMyKeyboardSafe = SafeKeyboardDetector.isCurrentKeyboardSafe()
  //...

>   const OnPressChangeKeyboard = SafeKeyboardDetector.showInputMethodPicker()

Reviewer: ```suggestion
  const onPressChangeKeyboard = SafeKeyboardDetector.showInputMethodPicker()
```

]
```

## Safe Keyboard Detector

⚠ Warning : This functionality is ONLY implemented on Android since iOS already prevent the use of third party keyboard on sensitive filed such as passwords. 

> **🥷 What's the threat?** Third party keyboard might embed a malicious keylogger program to record passwords and sensitive data [More details](https://www.synopsys.com/blogs/software-security/mitigate-third-party-mobile-keyboard-risk.html)

How to mitigate this threat : 

- Compare the current keyboard id with a list of known keyboard package names.
- Display the android native input method picker for the user to change its keyboard.

```tsx
import { SafeKeyboardDetector } from '@bam.tech/react-native-app-security';

> export default function App() {

Reviewer 2: Doesn't bring much to be in a component here
